include/mata/nft/nft.hh

```diff
@@ -51,6 +51,13 @@ public:
     using super::vector;
     Levels(const std::vector<Level>& levels): super(levels) {}
     Levels(std::vector<Level>&& levels): super(std::move(levels)) {}
+
+    /**
+     * @brief Append post vector to the delta.
+     *
+     * @param post_vector Vector of posts to be appended.
+     */
+    void append(const Levels& levels_vector) { for (const Level& level: levels_vector) { push_back(level); } }
```

Comment:
Shouldn't the name be `extend` instead? `Append` sounds like it only adds one element into the vector.

Reply:
You can append a container of elements as well. It is used. However, this name is copied over from `SparseSet` which already has this exact operation, so that you can write
```cpp
nft.initial.append(new_initial_states);
nft.delta.append(transitions_for_new_states);
nft.final.append(new_final_states);
nft.levels.append(new_levels_for_new_states);
```

Reply:
It's ok then.